components/Downloads/DownloadCards/DownloadCard/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { Tab } from 'react-tabs';
+import classNames from 'classnames';
+import { MdGetApp } from 'react-icons/md';
+import Image from 'next/image';
+import { useRouter } from 'next/router';
+import styles from './index.module.scss';
+import type { FC } from 'react';
+
+type DownloadCardProps = {
+  name: string;
+  icon: string;
+  label: string;
+  download: string;
+  filename: string;
+  selected: boolean;
+  onSelect: (name: string) => void;
```

Comment:
This upset ESLint because `name` is unused.
I tried finding how to resolve it, the solution asks for adding exception to the ESLint rules (e.g. ignore `_<variable-name>`, `_name`).

Reply:
```suggestion
  onSelect: (_name: string) => void;
```

Should do the trick :)

Reply:
Added the rule in this commit 066006a0ba03c595d7d484667995dc815d7365f1